CONTRIBUTING.md

```diff
@@ -18,6 +18,38 @@ It means that files ```EntityManagementEditor.tsx.ejs``` and ```EntityManagement
 
 > NOTE: This section is under construction.
 
+#### Cross-platforming
+
+When you working with code, you need to adhere to several recomendations for achive correct app working on UNIX and Windows OS:
+
+1. Remember, that Windows uses "\\" symbol as path separator, but UNIX uses "/". For this reason, you need to use "path" package, when you working with pathes in code:
```

Comment:
```suggestion
1. Remember that Windows uses "\" symbol as path separator, but UNIX uses "/". For this reason, you need to use "path" package when you work with paths in code:
```

Reply:
@vyacheslav-pushkin "\\" - this is done on purpose to show \ symbol in ""